src/main/resources/assets/js/launcher/main.ts

```diff
@@ -235,6 +230,20 @@ class Launcher {
         return `theme-${ThemeManager.getTheme(this.config['theme'])}`;
     };
 
+    private getLauncherUrl(): string {
+        const pathName = window.location.pathname;
+        const adminBaseUrl = this.config['adminBaseUrl'];
+
+        return pathName === `${adminBaseUrl}` || pathName === `${adminBaseUrl}/`
+                  ? UriHelper.joinPath(adminBaseUrl, `/${homeAppToolKey}/home/_/${homeAppToolKey}/launcher`)
```

Comment:
in theory access to `/com.enonic.xp.app.main/home/_/com.enonic.xp.app.main/launcher` should work even on dashboard

Reply:
homeAppToolKey = com.enonic.xp.app.main